database/psql.seed.js

```diff
@@ -73,16 +77,16 @@ csvStreamAuthors.pipe(writableStreamAuthors);
 csvStreamAuthors.on('drain', function() {
   writeAuthors();
 });
-var numOfAuthors =1000000;
+var numOfAuthors =1;
 
 function writeAuthors(){
 
-while( numOfAuthors>0){
-if(!csvStreamAuthors.write(generateAuthor())){
-  numOfAuthors--
+while( numOfAuthors<1000001){
+if(!csvStreamAuthors.write(generateAuthor(numOfAuthors))){
```

Comment:
I would advise linting your code w/ the Airbnb style guide so that your PR reviewers don't have to mention small details like "indent your code," "don't forget your semicolons," and "keep your spacing consistent"!